client/my-sites/plugins/plugin-ratings/index.jsx

```diff
@@ -48,26 +48,26 @@ export default React.createClass( {
 		const numberOfRatings = ( ratings && ratings[ ratingTier ] ) ? ratings[ ratingTier ] : 0;
 
 		return (
-			<div className="plugin-ratings__rating-tier" key={ `plugins-ratings__tier-${ ratingTier }` }>
-				<a className="plugin-ratings__rating-container" target="_blank" rel="noopener noreferrer"
-					onClick={ analytics.ga.recordEvent( 'Plugins', 'Clicked Plugin Ratings Link', 'Plugin Name', slug ) }
-					href={ this.buildReviewUrl( ratingTier ) }
-				>
-					<span className="plugin-ratings__rating-tier-text">
-						{
-							this.translate( '%(ratingTier)s stars', {
-								args: { ratingTier }
-							} )
-						}
-					</span>
-					<span className="plugin_ratings__bar">
-						<ProgressBar value={ numberOfRatings }
-							total={ numRatings }
-							title={ this.translate( '%(numberOfRatings)s ratings', { args: { numberOfRatings } } ) }
-						/>
-					</span>
-				</a>
-			</div>
+			<a key={ `plugins-ratings__tier-${ ratingTier }` } className="plugin-ratings__rating-container" target="_blank" rel="noopener noreferrer"
```

Comment:
Formatting wise, I think it would be more readable if we dropped each prop onto its own line instead of having `key`, `className`, etc. be on one line and then `onClick` and `href` be on separate lines.